app/controllers/admin/users_controller.rb

```diff
@@ -4,7 +4,19 @@ class UsersController < AdminController
     def close_account
       u = User.find(close_account_params)
       u.close_account!
-      redirect_to user_search_path, notice: t('admins.user_search.account_closing_scheduled', name: u.username)
+      redirect_to user_search_path, notice: t("admins.user_search.account_closing_scheduled", name: u.username)
+    end
+
+    def lock_account
+      u = User.find(close_account_params)
```

Comment:
Another detail I missed, let's actually refactor this and get rid of the rather unnecessary `close_account_params` here, passing `params[:id]` to `.find` directly is just fine. Please also update `close_account` for this and drop the method below.